frontend/pages/hosts/details/HostDetailsPage/modals/OSPolicyModal/OSPolicyModal.tsx

```diff
@@ -98,12 +94,12 @@ const RenderOSPolicyModal = ({
           value={osPolicy}
         />
         <div className="modal-cta-wrap">
-          <Button onClick={onCancel} variant="inverse">
-            Close
-          </Button>
           <Button onClick={onCreateNewPolicy} variant="brand">
             Create new policy
           </Button>
+          <Button onClick={onCancel} variant="inverse">
```

Comment:
These buttons were ordered wrong, brand button now correctly on right side of inverse button